app/views/devise/sessions/splash.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="top p-2">
+    <h2 class='text-light text-center h1'>Welcome</h2>
+</div>
+
+<div class="splash">
+    <div class='d-flex splash-footer p-3 justify-content-between gap-2'>
+      <%= link_to 'Sign In', new_user_session_path, class: 'btn border border-light p-2 text-light col-6' %>
+      <%= link_to 'Sign Up', new_user_registration_path, class: 'btn border border-light text-light col-6 p-2' %>
+    </div>
+</div>
```

Comment:
- [ ] For the splash screen, please make sure to follow the [design guidelines](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow%20these%20design%20guidelines%2C%20including%3A) for this page and other pages of the app. Also, please make sure to display the name of the app in the splash screen as required. Please take a look at the screens below.

![Screenshot (1)](https://github.com/alicemirigo92/Budget-App/assets/47625407/5dafaaca-7522-4189-ba89-090fd94a80d8)